package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/Hipo/stylelint-config-hipo-base#readme",
   "peerDependencies": {
-    "stylelint": "^10.1.0",
+    "stylelint": "11.0.0",
```

Comment:
For all the linters we use, I wonder, should we use carret `^` or tilde `~` instead of the exact versions? I feel like being flexible is more convenient. What do you think? 
This is also an issue for eslint plugins, too.

Reply:
I believe we can do that